### PR TITLE
Decode BasisU textures to BCn format instead of RGBA8

### PR DESCRIPTION
**Pull request by lilleyse**
Fixes https://github.com/CesiumGS/cesium-omniverse/issues/266

Previously, compressed textures were decoded to RGBA8 instead of the native BCn format due to a limitation in Kit 105.0. Now with Kit 105.1 we can decode to BCn and keep the data compressed on the GPU. This involves a bit of a strange workaround with the `stride` parameter (see comments in the code).

Test data:

[agi-ktx2-explicit.zip](https://github.com/CesiumGS/cesium-omniverse/files/11409647/agi-ktx2-explicit.zip)
[agi-ktx.usda.zip](https://github.com/CesiumGS/cesium-omniverse/files/11972717/agi-ktx.usda.zip)

### CODE REVIEW

**Comment by lilleyse** [comment]
I confirmed that the texture memory is lower because of this change. See the `Texture - Other` category. It's a small difference (112 MB - 101 MB), but I think it's because there's other textures being counted in that stat and the initial view doesn't load many tiles. I ran multiple runs and get the same result.

Before|After
--|--
before|after

File: src/core/src/Context.cpp [hunk 1]
```diff
@@ -637,6 +641,15 @@ const pxr::TfToken& Context::getCesiumMdlPathToken() const {
     return _cesiumMdlPathToken;
 }
 
+bool Context::isCompressedTexturesSupported() const {
+    // Compressed textures do not work in Kit 105.0 or 105.0.1
+    if (_kitVersion.rfind("105.0", 0) == 0) {
```
Comment (weegeekps) [inline src/core/src/Context.cpp:646]: Is this even necessary? We just released version 0.11.0 which claims that versions less than 105.1 are unsupported.

**Review by weegeekps — COMMENTED** [review]
One question but otherwise looks good.

File: src/core/src/Context.cpp (hunk 1 above)
Comment (lilleyse) [reply src/core/src/Context.cpp:646]: Oh good point... I'll remove it

Comment (lilleyse) [reply src/core/src/Context.cpp:646]: I think I worked on this PR before realizing that 0.11.0 wouldn't be backwards compatible with Kit 105.0

Comment (weegeekps) [reply src/core/src/Context.cpp:646]: In that case do we even need any of the code to pass the kit version up to the native layer?

Comment (lilleyse) [reply src/core/src/Context.cpp]: Nope, I just removed it all

**Review by weegeekps — APPROVED** [review]
Looks great and works great.

**Pull request merged**